src/bb-modules/Invoice/ServiceInvoiceItem.php

```diff
@@ -121,16 +121,8 @@ public function executeTask(\Model_InvoiceItem $item)
         }
 
         if (\Model_InvoiceItem::TYPE_DEPOSIT == $item->type) {
-            $clientService = $this->di['mod_service']('Client');
-
-            $invoice = $this->di['db']->getExistingModelById('Invoice', $item->invoice_id);
```

Comment:
Due to changes here the test seems to be failing

1) Box\Mod\Invoice\ServiceInvoiceItemTest::testexecuteTaskTypeDeposit
[44](https://github.com/FOSSBilling/FOSSBilling/actions/runs/3062942591/jobs/4949750766#step:10:45)
Expectation failed for method name is "getExistingModelById" when invoked at least once.
[45](https://github.com/FOSSBilling/FOSSBilling/actions/runs/3062942591/jobs/4949750766#step:10:46)
Expected invocation at least once but it never occurred.

It seems this function is used for automated testing to add "deposit" to the user

Reply:
Or we need to update ...